topo/node/juniper/juniper.go

```diff
@@ -130,7 +130,7 @@ func (n *Node) waitConfigInfraReadyAndPushConfigs(configs []string) error {
 	for time.Since(start) < configModeTimeout {
 		multiresp, err := n.cliConn.SendConfigs(configs)
```

Comment:
Also exploring - passing a higher timeout to SendConfigs as "options". The default might be low.

Reply:
ping need update on cla for that